CHANGELOG.rst

```diff
@@ -102,6 +103,7 @@
 
 .. _`traceback style docs`: https://pytest.org/latest/usage.html#modifying-python-traceback-printing
 
+.. _#469: https://github.com/pytest-dev/pytest/issues/469
```

Comment:
Moved in https://github.com/pytest-dev/pytest/pull/1432 plus a couple of comments on the changelog.